draft-ietf-quic-recovery.md

```diff
@@ -1067,14 +1066,18 @@ Pseudocode for OnLossDetectionTimeout follows:
 
 ~~~
 OnLossDetectionTimeout():
-  if (crypto packets are in flight):
+  packets_lost = false
+  for pn_space in [ Initial, Handshake, ApplicatonData ]:
+    if (loss_times[pn_space] != 0):
+      // Time threshold loss Detection
+      DetectLostPackets(pn_space)
```

Comment:
It seems a bit wasteful to run this for every packet number space that has a non-zero `loss_time`, since we'll only detect lost packets in the space that has the earliest `loss_time`.